js/views/playingScreen.js

```diff
@@ -141,14 +143,27 @@ var PlayingScreen = React.createClass({
   },
 
   handleTouchEnd: function(event) {
-    event.preventDefault();//to prevent mobile from propagating click to discovery shown on pause
-    if (!this.state.controlBarVisible){
-      this.showControlBar(event);
-      this.props.controller.startHideControlBarTimer();
-    }
-    else if (!this.props.controller.videoVr) {
-      this.props.controller.togglePlayPause(event);
+    if (event.target.className === "oo-state-screen-selectable") {
+      event.preventDefault();//to prevent mobile from propagating click to discovery shown on pause
+      if (!this.state.controlBarVisible){
+        this.showControlBar(event);
+        this.props.controller.startHideControlBarTimer();
+      }
+      else {
+        var isNeedToggle = false;
```

Comment:
Can you change this to var shouldToggle?

Reply:
\+